### PR TITLE
Mark Versioning documents as stable

### PR DESCRIPTION
**Pull request by tedsuo**
Fixes #
Resolves v1.0 blocker identified here: [](https://github.com/open-telemetry/opentelemetry-specification/pull/1372#issuecomment-775278062)

## Changes
* Marks "OpenTelemetry Environment Variable Specification" document as stable
* Marks "Versioning and stability for OpenTelemetry clients" document as stable

### CODE REVIEW

**Review by carlosalberto — APPROVED** [review]

**Review by cijothomas — APPROVED** [review]

**Comment by tedsuo** [comment]
Actually looks like this is redundant with https://github.com/open-telemetry/opentelemetry-specification/pull/1407

File: specification/sdk-environment-variables.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # OpenTelemetry Environment Variable Specification
 
+**Status**: [Stable](document-status.md)
```
Comment (bogdandrutu) [inline specification/sdk-environment-variables.md:3]: This is not stable correct? See comments in https://github.com/open-telemetry/opentelemetry-specification/pull/1407

**Review by bogdandrutu — CHANGES_REQUESTED** [review]

**Comment by bogdandrutu** [comment]
@tedsuo I missed the version document, we can make this one only for that.

File: specification/sdk-environment-variables.md (hunk 1 above)
Comment (bogdandrutu) [reply specification/sdk-environment-variables.md:3]: Consider to remove the env variables from this and make it only for versioning, so we can merge this quick :)

**Review by bogdandrutu — APPROVED** [review]

**Review by cijothomas — APPROVED** [review]

**Comment by bogdandrutu** [comment]
Trivial PR, we can merge fast :)

**Pull request merged**